### PR TITLE
KOGITO-3218 - fix data index infinispan jar

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/KOGITO-3218

This pr aims to fix the jenkins pipeline, using the data-index-storage-infinispan jar. 

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Kaitou786** [comment]
> /jenkins test

@r00ta there is some problem on jenkins side, on rhba side related to certificates which is being worked on right now. I'll manually start the pipeline when that is fixed :

**Comment by r00ta** [comment]
> > /jenkins test
> 
> @r00ta there is some problem on jenkins side, on rhba side related to certificates which is being worked on right now. I'll manually start the pipeline when that is fixed :

Hi @Kaitou786 , thank you!

**Comment by r00ta** [comment]
fyi I've opened another PR for the ticket https://github.com/kiegroup/kogito-images/pull/228
That one is using an env variable to choose which artifact to use. The current PR simply uses the infinispan one just to make jenkins happy and let you decide how to handle the two jars scenario in the future.

File: modules/kogito-data-index/module.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ version: "1.0.0-snapshot"
 
 artifacts:
   - name: kogito-data-index-runner.jar
-    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service/1.0.0-SNAPSHOT/data-index-service-1.0.0-20200805.111853-4-runner.jar
-    md5: fb97dc0981ed1bfb5d0689b3779dfbec
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-storage-infinispan/1.0.0-SNAPSHOT/data-index-storage-infinispan-1.0.0-20200831.024606-33.jar
```
Comment (danielezonca) [inline modules/kogito-data-index/module.yaml]: It is not _storage_ but _service_ (`data-index-service-infinispan`)

File: scripts/update-maven-artifacts.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 Modules = {
     #service-name: module-name(directory in which module's module.yaml file is present)
     #Note: Service name should be same as given in the repository
-    "data-index-service": "kogito-data-index",
+    "data-index-storage-infinispan": "kogito-data-index",
```
Comment (danielezonca) [inline scripts/update-maven-artifacts.py]: It is not _storage_ but _service_ (`data-index-service-infinispan`)

**Review by spolti — CHANGES_REQUESTED** [review]
please apply changes requested by Daniele.

**Pull request closed without merging**